src/main/java/io/github/foundationgames/automobility/Automobility.java

```diff
@@ -37,7 +37,7 @@ public void onInitialize() {
 
         AutomobilityData.setup();
 
-        ControllerUtils.initLCHandler();
+        ControllerUtils.initMCHandler();
```

Comment:
I think all instances of "MC" in relation to "MidnightControls" should be expanded out to "MidnightControls"

Reply:
Yeah, you're right, I did so in the latest commit.